### PR TITLE
Assume USSD is a valid channel + test

### PR DESCRIPTION
**Pull request by smn**
This addresses #499.

### CODE REVIEW

**Comment by smn** [comment]
@nicpottier & @rowanseymour ready for review. Not sure about the approach.

File: temba/channels/views.py [hunk 1]
```diff
@@ -527,6 +527,7 @@ def channel_status_processor(request):
         cutoff = timezone.now() - timedelta(hours=1)
         send_channel = org.get_send_channel(scheme=TEL_SCHEME)
         call_channel = org.get_call_channel()
+        ussd_channel = org.get_ussd_channel(scheme=TEL_SCHEME)
```
Comment (nicpottier) [inline temba/channels/views.py]: Why do we have scheme for `get_ussd_channel` anyways? Is there any scenario where that can't be `TEL`?

Comment (smn) [inline temba/channels/views.py]: right, not that I can think of. I'll refactor.

Comment (nicpottier) [inline temba/channels/views.py]: Awesome, thanks!

**Comment by smn** [comment]
@nicpottier ready for review again

**Pull request merged**